README.md

```diff
@@ -153,10 +165,7 @@ def run_coroutine_with_span(span, func, *args, **kwargs):
     :param args: Positional args to func, if any.
     :param kwargs: Keyword args to func, if any.
     """
-    def mgr():
-        return RequestContextManager(span)
-
-    with tornado.stack_context.StackContext(mgr):
```

Comment:
We are using internally now a subclass of `StackContext`, so contexts will be properly stashed/restored, but a few details will be different. Everything should work if the user uses the `span_in_stack_context()` function, though. But definitely needs clarification, just in case. Will add it 👍